typedb/driver/TypeDBSession.java

```diff
@@ -18,22 +18,34 @@
 package com.vaticle.typedb.benchmark.typedb.driver;
 
 import com.vaticle.typedb.benchmark.simulation.driver.Session;
+import com.vaticle.typedb.client.api.connection.TypeDBOptions;
 
-import static com.vaticle.typedb.client.api.TypeDBTransaction.Type.WRITE;
+import static com.vaticle.typedb.client.api.connection.TypeDBTransaction.Type.READ;
+import static com.vaticle.typedb.client.api.connection.TypeDBTransaction.Type.WRITE;
 
 public class TypeDBSession implements Session<TypeDBTransaction> {
 
-    private final com.vaticle.typedb.client.api.TypeDBSession nativeSession;
+    private final com.vaticle.typedb.client.api.connection.TypeDBSession nativeSession;
```

Comment:
We've reverted this change in typedb-client-java. Let's upgrade and revert this change too.

Reply:
I've upgraded but I don't see this change reverted in `typedb-client-java`!